src/braket/experimental/algorithms/adaptive_shot_allocation/adaptive_shot_allocation.md

```diff
@@ -0,0 +1,7 @@
+Properly allocating shots to different terms of a Hamiltonian when calculating expectation values can drasticly improve the accuracy of the estimator. A generally adopted strategy is to allocate shots proportionally to the magnitude of a term's coefficient. However, in certain situations that strategy can severely underperform. The code here implements an adaptive shot-allocation algorithm from (see "Adaptive Estimation of Quantum Observables" by Shlosberg et al, DOI: 10.22331/q-2023-01-26-906), and the accompanying notebooks illustrate the benefits of using it.
+
+<!--
+[metadata-name]: Adaptive Shot Allocation
+[metadata-tags]: Textbook
```

Comment:
This is inconsistent with the notebook folder which is under `advanced_algorithms`

Reply:
Can you also add a row to the table in the README.md at the top level?

Reply:
Also, do you know where does the content "adaptive_shot_allocation.md" go to? It has a link so I assume it would got embedded somewhere, but I didn't find it.